module_3/Service/src/main/java/com/epam/ems/service/user/UserService.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.ems.service.user;
+
+import com.epam.ems.dao.CRUDDao;
+import com.epam.ems.dao.orderdao.OrderDaoImpl;
+import com.epam.ems.dao.userdao.UserDaoImpl;
+import com.epam.ems.dto.Certificate;
+import com.epam.ems.dto.Tag;
+import com.epam.ems.dto.User;
+import com.epam.ems.dto.UserOrderInfo;
+import com.epam.ems.exceptions.NotEnoughMoneyException;
+import com.epam.ems.logic.handler.DateHandler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class UserService {
+
+    @Autowired
+    private UserDaoImpl dao;
+
+    @Autowired
+    private CRUDDao<Certificate> certificateDao;
+
+    @Autowired
+    private OrderDaoImpl orderDao;
+
+    @Autowired
+    private DateHandler dateHandler;
```

Comment:
No need in this resource - the required method is static.
Code convention says if we have a static method we have to call it using class, e.g. DateHandler.staticMethod();